mitodl/mitxpro
# For Teams Section (#148)

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes (#148 )

#### What's this PR do?
Adds the "For Teams" section to course pages

#### How should this be manually tested?
Add a "For Teams Page" type subpage to a course page. The section should appear with the expected design and content.

#### Where should the reviewer start?
Start by adding a "For Teams Page" type subpage to a course page as advised above.

#### Screenshots (if appropriate)
![Screenshot from 2019-04-30 18-26-46](https://user-images.githubusercontent.com/45350418/56965129-ff4b9580-6b75-11e9-9391-6652878d9086.png)
![Screenshot from 2019-04-30 18-28-00](https://user-images.githubusercontent.com/45350418/56965138-0377b300-6b76-11e9-8598-c7480bbf74f0.png)
![screeny-for-teams](https://user-images.githubusercontent.com/45350418/56965144-070b3a00-6b76-11e9-9049-3b139c3e726d.png)
